pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[project]
+name = "OpenImageIO"
+version = "2.4.0.dev1"
```

Comment:
Yes, I updated with 2.6.0.2 because that looked reasonably close. It doesn't really matter yet because even after this is merged, and we can see GitHub actions building wheels, we'll need to [integrate GitHub Actions with Trusted Publishers](https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/). Or in the short term, if you wanted to publish to PyPI before that happens, we could make sure the version number is correct in the release branch then [manually upload wheels with twine](https://twine.readthedocs.io/en/stable/).